src/services/project-user-service.js

```diff
@@ -11,3 +11,28 @@ export async function getUsersInProjects(projectIds) {
   )
   return rows
 }
+
+export async function getMembersInProject(projectId) {
```

Comment:
Alright, I implemented the suggested changes in the latest commit. I originally wanted to avoid touching the getUsersInProjects method because it wasn't mine and I did not want to impact other code.